Dockerfile.builder

```diff
@@ -18,13 +19,13 @@ RUN yum --enablerepo=centosplus install -y --quiet \
       postgresql \
     && yum clean all
 
-RUN test -n $USE_GO_VERSION_FROM_WEBSITE \
-    && cd /tmp \
+RUN if [[ "$USE_GO_VERSION_FROM_WEBSITE" = 1 ]]; then cd /tmp \
     && wget --no-verbose https://dl.google.com/go/go1.10.linux-amd64.tar.gz \
     && echo "b5a64335f1490277b585832d1f6c7f8c6c11206cba5cd3f771dcb87b98ad1a33  go1.10.linux-amd64.tar.gz" > checksum \
     && sha256sum -c checksum \
     && tar -C /usr/local -xzf go1.10.linux-amd64.tar.gz \
-    && rm -f go1.10.linux-amd64.tar.gz
+    && rm -f go1.10.linux-amd64.tar.gz; \
```

Comment:
Could we have 1.11 here as well?

Reply:
Done in https://github.com/fabric8-services/fabric8-wit/pull/2378/commits/f7555124e5f096a4a3dfc909b54c5c8724a4527d

Reply:
This discussion is out of date. I've made some changes to the PR.